### PR TITLE
[CAS-691] Review tutorial dependencies

### PR DESCRIPTION
**Pull request by bychkovdmitry**
Review tutorial dependencies

### CODE REVIEW

File: samplejava/build.gradle [hunk 1]
```diff
@@ -36,12 +34,8 @@ android {
...
-    implementation 'io.coil-kt:coil:1.1.1'
 
-    implementation 'androidx.appcompat:appcompat:1.2.0'
+    implementation 'io.coil-kt:coil:1.1.1'
     implementation 'com.google.android.material:material:1.3.0'
-    implementation 'androidx.constraintlayout:constraintlayout:2.0.4'
-    testImplementation 'junit:junit:4.+'
-    androidTestImplementation 'androidx.test.ext:junit:1.1.2'
-    androidTestImplementation 'androidx.test.espresso:espresso-core:3.3.0'
+    implementation "androidx.fragment:fragment-ktx:1.3.0-rc02"
```
Comment (zsmb13) [inline samplejava/build.gradle]: `1.3.0` is now available as stable

Comment (bychkovdmitry) [reply samplejava/build.gradle]: Done

File: samplejava/build.gradle [hunk 2]
```diff
@@ -36,12 +34,8 @@ android {
...
     implementation "io.getstream:stream-chat-android-ui-components:4.5.4"
-    implementation 'io.coil-kt:coil:1.1.1'
 
-    implementation 'androidx.appcompat:appcompat:1.2.0'
+    implementation 'io.coil-kt:coil:1.1.1'
     implementation 'com.google.android.material:material:1.3.0'
-    implementation 'androidx.constraintlayout:constraintlayout:2.0.4'
-    testImplementation 'junit:junit:4.+'
-    androidTestImplementation 'androidx.test.ext:junit:1.1.2'
-    androidTestImplementation 'androidx.test.espresso:espresso-core:3.3.0'
```
Comment (zsmb13) [inline samplejava/build.gradle]: Let's keep all the default dependencies in here, and just add the new ones in above as required

```
// Add new dependencies
<new deps we add>

<original deps>
```

**Review by zsmb13 — CHANGES_REQUESTED** [review]

**Comment by bychkovdmitry** [comment]
Created 2 empty projects (java and kotlin) and verified the default dependencies below empty line. Also, tested locally published library from this branch: https://github.com/GetStream/stream-chat-android/tree/feature/temp-dep-test-publishing and had to add one extra dependency.

File: samplejava/build.gradle (hunk 2 above)
Comment (bychkovdmitry) [reply samplejava/build.gradle]: Sorry, I finally got the idea

**Review by zsmb13 — APPROVED** [review]

**Pull request merged**